Source/WTF/wtf/SystemTracing.h

```diff
@@ -173,10 +180,445 @@ enum TracePointCode {
+        case BuildTransactionStart:
+        case BuildTransactionEnd:
+            return "BuildTransaction";
+        case SyncMessageStart:
+        case SyncMessageEnd:
+            return "SyncMessage";
+        case SyncTouchEventStart:
+        case SyncTouchEventEnd:
+            return "SyncTouchEvent";
+        case InitializeWebProcessStart:
+        case InitializeWebProcessEnd:
+            return "InitializeWebProcess";
+        case RenderingUpdateRunLoopObserverStart:
+        case RenderingUpdateRunLoopObserverEnd:
+            return "RenderingUpdateRunLoopObserver";
+        case LayerTreeFreezeStart:
+        case LayerTreeFreezeEnd:
+            return "LayerTreeFreeze";
+        case FlushRemoteImageBufferStart:
+        case FlushRemoteImageBufferEnd:
+            return "FlushRemoteImageBuffer";
+        case CreateInjectedBundleStart:
+        case CreateInjectedBundleEnd:
+            return "CreateInjectedBundle";
+        case PaintSnapshotStart:
+        case PaintSnapshotEnd:
+            return "PaintSnapshot";
+        case RenderServerSnapshotStart:
+        case RenderServerSnapshotEnd:
+            return "RenderServerSnapshot";
+        case TakeSnapshotStart:
+        case TakeSnapshotEnd:
+            return "TakeSnapshot";
+        case SyntheticMomentumStart:
+        case SyntheticMomentumEnd:
+            return "SyntheticMomentum";
+        case SyntheticMomentumEvent:
+            return "SyntheticMomentumEvent";
+        case RemoteLayerTreeScheduleRenderingUpdate:
+            return "RemoteLayerTreeScheduleRenderingUpdate";
+
+        case CommitLayerTreeStart:
+        case CommitLayerTreeEnd:
+            return "CommitLayerTree";
+        case ProcessLaunchStart:
+        case ProcessLaunchEnd:
+            return "ProcessLaunch";
+        case InitializeSandboxStart:
+        case InitializeSandboxEnd:
+            return "InitializeSandbox";
+        case WebXRCPFrameWaitStart:
+        case WebXRCPFrameWaitEnd:
+            return "WebXRCPFrameWait";
+        case WebXRCPFrameStartSubmissionStart:
+        case WebXRCPFrameStartSubmissionEnd:
+            return "WebXRCPFrameStartSubmission";
+        case WebXRCPFrameEndSubmissionStart:
+        case WebXRCPFrameEndSubmissionEnd:
+            return "WebXRCPFrameEndSubmission";
+
+        case WakeUpAndApplyDisplayListStart:
+        case WakeUpAndApplyDisplayListEnd:
+            return "WakeUpAndApplyDisplayList";
+
+
+        case WTFRange:
+        case JavaScriptRange:
+        case WebCoreRange:
+        case WebKitRange:
+        case WebKit2Range:
+        case UIProcessRange:
+        case GPUProcessRange:
+            return nullptr;
+        }
+    }
+
+    std::string processName { "WebKit" };
+    static const char* getProcessName()
+    {
+        return getInstance().processName.c_str();
+    }
+
+    SysprofAnnotator() { };
+    ~SysprofAnnotator()
+    {
+        for (auto const & iter : ongoingMarks) {
```

Comment:
I think the style checker will want you to remove the spaces at the left of the ampersand, and also we typically see `const auto` more often than `auto const` in WebKit code:

 ```suggestion
         for (const auto& iter : ongoingMarks) {
```